addons/jest/src/index.js

```diff
@@ -1,6 +1,6 @@
 import addons from '@storybook/addons';
 
-const findTestResults = (testFiles, jestTestResults, jestTestFilesExt) =>
+const findTestResults = (testFiles, jestTestResults, jestTestFilesExt = "") =>
```

Comment:
I wonder did you run `git commit --no-verify` instead of just `git commit`? We have a git hook which runs eslint in "fix" mode, which would replace those double quotes with single ones

Reply:
Likely didn't do `yarn install`, it's OK, we'll do a fix commit soon.

Reply:
Ok, I'll run `yarn lint:js --fix` after merging

Reply:
Yikes, noticed the double quotes after commiting, thanks!